### PR TITLE
Web calendar-styled via guide. 

### PR DESCRIPTION
**Pull request by bykristea**
# Description

Found way to style Calendar on Landing Page.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: web-workout-tracker/workout/package_REMOTE_87276.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "workout",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^1.1.1",
+    "auth0-js": "^9.11.3",
+    "auth0-lock": "^11.17.2",
+    "axios": "^0.19.0",
+<<<<<<< Updated upstream
```
Comment (nia-ja) [inline web-workout-tracker/workout/package_REMOTE_87276.json]: It shouldn't be there. Looks like not resolved merge conflict

File: web-workout-tracker/workout/package_REMOTE_87276.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+=======
+    "less": "^3.10.3",
+    "moment": "^2.24.0",
+>>>>>>> Stashed changes
```
Comment (nia-ja) [inline web-workout-tracker/workout/package_REMOTE_87276.json]: same

File: web-workout-tracker/workout/package_BACKUP_87276.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "workout",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^1.1.1",
+    "auth0-js": "^9.11.3",
+    "auth0-lock": "^11.17.2",
+    "axios": "^0.19.0",
+<<<<<<< HEAD
```
Comment (nia-ja) [inline web-workout-tracker/workout/package_BACKUP_87276.json]: Shouldn't be there either

File: web-workout-tracker/workout/package_REMOTE_87276.json (hunk 1 above)
Comment (bykristea) [reply web-workout-tracker/workout/package_REMOTE_87276.json]: done!

File: web-workout-tracker/workout/package_REMOTE_87276.json (hunk 2 above)
Comment (bykristea) [reply web-workout-tracker/workout/package_REMOTE_87276.json]: done!

**Pull request merged**